engine/crates/integration-tests/tests/federation/introspection.rs

```diff
@@ -819,7 +819,6 @@ fn introspection_on_multiple_federation_subgraphs() {
     }
 
     type Picture {
-      altText: String!
```

Comment:
This one is `@inaccessible`, so it's expected that it doesn't show up in introspection anymore.